packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -44,26 +48,6 @@ import type {
 
 const DEFAULT_INTERVAL = 180000;
 
-/**
- * Compare 2 given strings and return boolean
- * eg: "foo" and "FOO" => true
- * eg: "foo" and "bar" => false
- * eg: "foo" and 123 => false
- *
- * @param value1 - first string to compare
- * @param value2 - first string to compare
- * @returns true if 2 strings are identical when they are lowercase
- */
-export function isEqualCaseInsensitive(
```

Comment:
Note, this is not exported outside of the assets-controller package, and therefore does not require a changelog item